MAUI/Maps/zoom-pan.md

```diff
@@ -0,0 +1,633 @@
+</ContentPage.Resources>
+
+<maps:SfMaps>
+    <maps:SfMaps.Layer>
+        <maps:MapTileLayer Radius="5"
+                           DistanceType="Kilometer"
+                           MarkerTemplate="{StaticResource MapMarkerTemplate}"
+                           UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png">
+            
+            <maps:MapTileLayer.Center>
+                <maps:MapLatLng Latitude="38.909804"
+                                Longitude="-77.043442">
+                </maps:MapLatLng>
+            </maps:MapTileLayer.Center>
+            
+            <maps:MapTileLayer.ZoomPanBehavior>
+                <maps:MapZoomPanBehavior ZoomLevel="1" />
+            </maps:MapTileLayer.ZoomPanBehavior>
+            
+            <maps:MapTileLayer.Markers>
+                <maps:MapMarkerCollection>
+                    <maps:MapMarker Latitude="38.909804"
+                                    Longitude="-77.043442" />
+                </maps:MapMarkerCollection>
+            </maps:MapTileLayer.Markers>
+        </maps:MapTileLayer>
+    </maps:SfMaps.Layer>
+</maps:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+public LatLngBoundsPage()
+{
+	InitializeComponent();
+    SfMaps map = new SfMaps();
+    MapTileLayer tileLayer = new MapTileLayer();
+    tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+    tileLayer.Radius = 5;
+    tileLayer.DistanceType = MapDistanceType.Kilometer;
+    MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+    zoomPanBehavior.ZoomLevel = 1;
+    tileLayer.Center = new MapLatLng(38.909804, -77.043442);
+    tileLayer.ZoomPanBehavior = zoomPanBehavior;
+    MapMarker mapMarker = new MapMarker();
+    mapMarker.Latitude = 38.909804;
+    mapMarker.Longitude = -77.043442;
+    MapMarkerCollection mapMarkers = new MapMarkerCollection();
+    mapMarkers.Add(mapMarker);
+    tileLayer.Markers = mapMarkers;
+    tileLayer.MarkerTemplate = CreateDataTemplate();
+    map.Layer = tileLayer;
+    this.Content = map;
+}
+private DataTemplate CreateDataTemplate()
+{
+    return new DataTemplate(() =>
+    {
+        var stackLayout = new StackLayout();
+        stackLayout.HorizontalOptions = LayoutOptions.StartAndExpand;
+        stackLayout.VerticalOptions = LayoutOptions.Center;
+        var image = new Image
+        {
+            Source = "map.png",
+            Scale =1,
+            Aspect = Aspect.AspectFit,
+            WidthRequest = 25,
+            HeightRequest = 35,
+            HorizontalOptions = LayoutOptions.Center,
+            VerticalOptions = LayoutOptions.Center
+        };
+        stackLayout.Add(image);
+        return new ViewCell { View = stackLayout };
+    });
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![SfMaps zoom level changed image](images/zoom-pan/distance_radius.png)
+
+### Geo-bounds
+
+Calculate the zoom level automatically based on the `MapLatLngBounds(Northeast, Southwest)` of `MapTileLayer` class.
```

Comment:
Calculate the zoom level automatically based on the `MapLatLngBounds(Northeast, Southwest)` of the `MapTileLayer` class.

Reply:
Modified the content